mediacapture-streams/MediaDevices-enumerateDevices-returned-objects.https.html

```diff
@@ -13,8 +13,10 @@
 function doTest(callGetUserMedia, testName)
 {
     promise_test(async () => {
-        if (callGetUserMedia) {
-            await setMediaPermission();
+        if (!callGetUserMedia) {
+            await setMediaPermission("denied");
```

Comment:
This test is testing the wrong thing / is too permissive.

enumerateDevices exposure is not tied to permission, but to [device-information-can-be-exposed](https://w3c.github.io/mediacapture-main/getusermedia.html#device-information-can-be-exposed), i.e. [[[canExposeCameraInfo](https://w3c.github.io/mediacapture-main/getusermedia.html#dfn-canexposecamerainfo)]] OR [[[canExposeMicrophoneInfo](https://w3c.github.io/mediacapture-main/getusermedia.html#dfn-canexposemicrophoneinfo)]] OR live-checks.

Specifically, we need to test that information is not exposed in cases where persistent permission is granted, but getUserMedia has not been called from this document before. See steps in [crbug 1101860](https://crbug.com/1101860).

Reply:
I've addressed the permission-coupling in 043b8f1 and proposed an additional test in dce98c1  to cater for the persistent permission situation - PTAL